src/client/client.h

```diff
@@ -440,10 +440,13 @@ class Client : public con::PeerHandler, public InventoryManager, public IGameDef
 	{
 		return m_env.getLocalPlayer()->formspec_prepend;
 	}
+
 	inline MeshGrid getMeshGrid()
 	{
 		return m_mesh_grid;
 	}
+  
+  bool m_inhibit_inventory_revert = false;
```

Comment:
This variable needs a good description of what it does. Furthermore, consider dropping the `m_` prefix. This is not a private member.